src/common/components/header/header.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import '../../../theme/topbar.css';
```

Comment:
My fault. 
When we have internal imports and third party imports we put the third party at the top and seperate them from the internal by adding a space
```suggestion
import React, { useState } from 'react';

import '../../../theme/topbar.css';
```